submissions/NastjonkaK/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,92 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {
+  type: "dog",
+  name: "Charlie",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  hasTail: true,
+  saying: "woof-woof"
+};
+
+const cat = {
+  type: "cat",
+  name: "Luna",
+  gender: "female",
+  legs: 4,
+  hands: 0,
+  hasTail: true,
+  saying: "meow-meow"
+};
+
+const man = {
+  type: "human",
+  name: "Max",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  hasTail: false,
+  saying: "Hello everybody!"
+};
+
+const woman = {
+  type: "human",
+  name: "Emma",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  hasTail: false,
+  saying: "Have a nice day!"
+};
+
+const womanCat = {
+  type: "human-cat",
+  name: "Phoebe",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  hasTail: true,
+  saying: cat.saying
+};
+
+function objToString(obj) {
+  let str = "";
+  for (let p in obj) {
+    if (obj.hasOwnProperty(p)) {
+      str += p + ": " + obj[p] + "; ";
+    }
+  }
```

Comment:
Consider employing [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) method. In combination with `#Array.map` and `#Array.join` it would make this fragment of code shorter and semantically cleaner.

Reply:
Thanks for your comments. I used the methods you wrote above.